### PR TITLE
fix: Malformed name of aab file

### LINKED ISSUE #2064: Malformed name of aab file
aab file has wrong name - eventyay-attendee-dev-aab

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #2064

### CODE REVIEW

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -45,6 +45,12 @@ if [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" ]; then
 	done
 fi
 
+if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+	mv eventyay-attendee-master-aab eventyay-attendee-master.aab
+else
+	mv eventyay-attendee-dev-aab eventyay-attendee-dev.aab
+fi
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: This is not the right solution

**Comment by iamareebjamal** [comment]
Also, on merge of master, every file should be deleted or else, the old named files will never be deleted

**Comment by liveHarshit** [comment]
> Also, on merge of master, every file should be deleted

So it should be - 
```bash
if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
	/bin/rm -f  *
else
	/bin/rm -f open-event-attendee-dev-*
fi
```

**Comment by iamareebjamal** [comment]
Yes

**Comment by liveHarshit** [comment]
> Yes

Updated

**Pull request merged**